addons/hr_holidays/models/hr_employee.py

```diff
@@ -179,7 +179,8 @@ def _search_absent_employee(self, operator, value):
             ('date_from', '<=', today_end),
             ('date_to', '>=', today_start),
         ])
-        return [('id', 'in', holidays.mapped('employee_id').ids)]
+        op = 'in' if operator == '=' else 'not in'
```

Comment:
```suggestion
        if (operator == '=' and value) or (operator == '!=' and not value):
            op = 'in'
        else:
            op = 'not in'
```

if we implement other things than "('is_absent', '=', True)" we might as well take all possibility (for boolean search) into account